.github/workflows/coverage-reports.yml

```diff
@@ -0,0 +1,118 @@
+# ------------------------------------------------------------
+# Copyright 2021 The Dapr Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+name: Publishing test coverage reports of components
+
+on:
+  repository_dispatch:
+    types: [coverage-report]
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */24 * * *'
```

Comment:
Instead of every 24 hours i.e. daily, we may want this report every 3 or 7 days, something like: "0 3 * * 2,5" (at 3 A.M. on Tuesday and Friday).

Reply:
I too agree that once a week is good enough.

Reply:
Done